front/scripts/i18n-api-errors.ts

```diff
@@ -65,19 +66,21 @@ async function checkI18N(error: any): Promise<string[]> {
 
 async function run() {
   try {
-    const api = await SwaggerParser.validate(openapi_path);
+    const api: OpenAPI.Document = await SwaggerParser.validate(openapi_path);
+
+    if (!('openapi' in api)) throw new Error('Expected an OpenAPI schema');
 
     // Do some checks on the generic error
-    if (!api.components.schemas.EditoastError)
-      throw new Error(`"EditoastError" can't be found in "components > schemas"`);
-    if (!api.components.schemas.EditoastError.oneOf)
+    const EditoastError = api?.components?.schemas?.EditoastError;
```

Comment:
Would you consider to use a small case for this const ?

```suggestion
    const editoastError = api?.components?.schemas?.EditoastError;
```

Reply:
Sure. This represents a schema type so I originally just kept the uppercase letter, but using a lowercase letter is fine by me as well.